android/src/app/organicmaps/util/Utils.java

```diff
@@ -810,4 +811,29 @@ public void onCompleted(final boolean success, @Nullable final String zipPath)
       });
     }
   }
+
+  @Nullable
+  public static <T> T getParcelable(Bundle args, String key, Class<T> clazz)
+  {
+    args.setClassLoader(clazz.getClassLoader());
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU)
+    {
+      return args.getParcelable(key, clazz);
+    }
+    return getParcelableOld(args, key, clazz);
+  }
+
+  @SuppressWarnings("deprecation")
```

Comment:
Is deprecation suppression really needed if you clearly call this API inside the if (SDK LEVEL) clause?

Reply:
Yes. Lint still produces deprecation warnings during compilation

Reply:
Strangely, it worked previously in Android Studio. Did you try to move this code inside the if in the other function?

Reply:
No. Maybe you didn't pay attention on build output tab. There is no way to tell the compiler "it's ok, don't show that this method is deprecated, we're using it only on older APIs". We compile our code with API 33 where this method annotated with `@Deprecated`, so the only solution is to suppress such warnings (at least I didn't find anything suitable). That's weird, but that how it works.

<img width="583" alt="image" src="https://user-images.githubusercontent.com/10351358/199107022-e38ea5db-ffec-4c9e-a1e5-5e238cc9df55.png">

```
organicmaps/android/src/app/organicmaps/util/Utils.java:822: warning: [deprecation] <T>getParcelable(String) in Bundle has been deprecated
      args.getParcelable(key);
          ^
  where T is a type-variable:
    T extends Parcelable declared in method <T>getParcelable(String)
```

Reply:
The older linter was smarter...